### DISCUSSION TITLE
Export only 20k

### DISCUSSION THREAD

**Discussion by ArniSamano** [YetiForceCompany/YetiForceCRM#17101] [How to] [answered]
Hi Admins

I have 100k records in module but when exporting it exports only 20k. I guess I missing something?
Help appreciated

**Comment by rskrzypczak** [comment, marked as answer]
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/10221999/e2dcb64e-b3cb-41b9-9c95-7800e281e6a6)
or
https://github.com/YetiForceCompany/YetiForceCRM/blob/d4b74a8ccd9696c06e5e8799655c14d1964cdd4e/config/Performance.php#L53-L54
